src/ovp_pipeline/commands/backfill_entities.py

```
@@ -0,0 +1,209 @@
+        "files_processed": processed,
+        "errors": errors,
+        "mentions_extracted": total_mentions,
+        "candidates_created": total_candidates,
+        "registry_before": before_count,
+        "registry_after": after_count,
+        "new_entities": after_count - before_count,
+        "elapsed_seconds": round(elapsed, 1),
+        "confidence_threshold": confidence_threshold,
+    }
+
+    print(f"\nDone.")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)